content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/10-migrate-vm-to-different-node.md

```diff
@@ -14,67 +14,134 @@ import InfoBox from 'shared/components/InfoBox';
+3. Navigate to  the **Virtual Machines** tab, and click the VM you migrated. 
 
 
-4. From the **three-dotMenu** or the **Actions drop-down Menu** for a selected VM, click **Migrate Node to Node**.
+4. Click the **Details** tab, and verify that the name and IP address of the new node is changed.
 
 
-# Validation
+# Maintain a Node
 
-1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+Compute nodes can be placed into maintenance mode using the `cordon` and `drain` commands. These commands mark the node as un-schedulable and drains all the virtual machines and pods from it. This process is useful in case you need to perform hardware maintenance on the node - for example to replace a disk or network interface card (NIC) card, perform memory maintenance, or if there are any issues with a particular node that need to be resolved. To learn more, check out the [Safely Drain a Node](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/#use-kubectl-drain-to-remove-a-node-from-service) Kubernetes resource.  
+
+
+## Prerequisites
+
+- Ensure `LiveMigrate` is set as the eviction strategy for all affected VMs. This provides seamless migration when the host is put in maintenance mode.   
+
+
+## Manually Migrate the VM
+
+<br />
 
+1. Obtain the kubeconfig file from Palette, and set the KUBECONFIG environment variable to access it so you can issue kubectl commands to the cluster. To learn how, refer to [Set up Kubectl](https://docs.spectrocloud.com/clusters/cluster-management/palette-webctl/#setupkubectl).
 
-2. From the left **Main Menu**, click **Clusters** and click on your cluster. 
 
+2. Issue the following command to mark the node as *un-schedulable*. This alerts the Kubernetes scheduler not to schedule any new pods on that node but allows existing pods running on the node to continue to run.
+    
+   Example:
+    ```bash
+    kubectl cordon <node-name>
+    ``` 
+    
+    **node-name**: The name of the node that should be marked as *un-schedulable*.
 
-3. Navigate to  the **Virtual Machines** tab, and click the VM that you migrated. 
 
+3. Issue the following command to gracefully remove all pods from the node that is undergoing maintenance. When you drain a node, all pods and VMs will be safely evicted from the node.
 
-4. Click the **Details** tab.
+    Example:
+    ```bash
+    kubectl drain <node-name>
+    ```
 
+    **node-name**: The name of the node that you wish to drain.
+    
+    <br />
+    
+    <InfoBox>
 
-5. Verify the name and IP address of the new node is changed.
+    The kubectl `drain` command should only be issued to a single node at a time.
 
+    </InfoBox>
 
 
+## Validation
 
 
+1. Using kubectl, log in to a machine that has access to the kubernetes cluster. 
+
+
+2. Issue the following command to verify the pods are rescheduled on a different node by verifying the name and IP address of the new node changed.
+
+    
+    ```bash
+    kubectl get pods -o wide
```

Comment:
```suggestion
    kubectl get pods --output wide
```
Always default to the long form for a command. That way the user can better understand what the command is doing.